From 78edea4f18332fd703cb343360b77c496558fd2d Mon Sep 17 00:00:00 2001
From: Guanhua Gao <guanhua.gao@nxp.com>
Date: Wed, 30 May 2018 18:16:56 +0800
Subject: [PATCH 533/767] Revert "net: Use TC_SETUP_QDISC_MQPRIO"

Most drivers still use TC_SETUP_MQPRIO, so revert the patch.

Signed-off-by: Guanhua Gao <guanhua.gao@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/net/ethernet/amd/xgbe/xgbe-drv.c | 2 +-
 include/linux/netdevice.h                | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/amd/xgbe/xgbe-drv.c b/drivers/net/ethernet/amd/xgbe/xgbe-drv.c
index 24f1053b8785..093e2fd1208b 100644
--- a/drivers/net/ethernet/amd/xgbe/xgbe-drv.c
+++ b/drivers/net/ethernet/amd/xgbe/xgbe-drv.c
@@ -2253,7 +2253,7 @@ static int xgbe_setup_tc(struct net_device *netdev, enum tc_setup_type type,
 	struct tc_mqprio_qopt *mqprio = type_data;
 	u8 tc;
 
-	if (type != TC_SETUP_QDISC_MQPRIO)
+	if (type != TC_SETUP_MQPRIO)
 		return -EOPNOTSUPP;
 
 	mqprio->hw = TC_MQPRIO_HW_OFFLOAD_TCS;
diff --git a/include/linux/netdevice.h b/include/linux/netdevice.h
index 97ba8c46040b..dfe0456ac873 100644
--- a/include/linux/netdevice.h
+++ b/include/linux/netdevice.h
@@ -798,7 +798,7 @@ typedef u16 (*select_queue_fallback_t)(struct net_device *dev,
 				       struct sk_buff *skb);
 
 enum tc_setup_type {
-	TC_SETUP_QDISC_MQPRIO,
+	TC_SETUP_MQPRIO,
 	TC_SETUP_CLSU32,
 	TC_SETUP_CLSFLOWER,
 	TC_SETUP_CLSMATCHALL,
-- 
2.17.0

